jballerina.java.arrays-ballerina/jarray_utils.bal

```diff
@@ -20,7 +20,7 @@ import ballerina/jballerina.java;
 # with a `panic` if the specified handle refers to a Java null or if zero dimensions have been provided.
 # ```ballerina
 # handle stringClass = check java:getClass("java.lang.String");
-# handle StrArray = java.arrays:newInstance(stringClass, 4);
+# handle StrArray = jballerina.java.arrays:newInstance(stringClass, 4);
```

Comment:
is this correct? can't we just say `handle StrArray = arrays:newInstance(stringClass, 4);`